share/fr.po

```diff
@@ -73,5 +34,39 @@ msgstr "Le type d'empreinte doit être un entier positif"
 msgid "Keytag must be a positive integer"
 msgstr "L'indentifiant doit être un entier positif"
 
+msgid "Domain name required"
+msgstr "Nom de domaine requis"
+
+msgid "The domain name is IDNA invalid"
+msgstr "Le nom de domaine n'est pas un domain IDNA valide"
+
+msgid ""
+"The domain name contains non-ascii characters and IDNA support is not "
+"installed"
+msgstr ""
+"Le nom de domaine contient des caractères non ascii et le support d'IDNA "
+"n'est pas installé"
+
+msgid "The domain name character(s) are not supported"
+msgstr "Les caractères du nom de domaine ne sont pas supportés"
+
+msgid "The domain name or label is too long"
+msgstr "Le nom de domaine ou les labels sont trop longs"
+
 msgid "Invalid language tag format"
 msgstr "Format de l'étiquette d'indentification de langue incorrect"
+
+msgid "Unkown language string"
+msgstr "Étiquette d'indentification de langue inconnue"
+
+msgid "Language string not unique"
+msgstr "Étiquette d'indentification de langue non unique"
+
+msgid "Invalid IP address"
+msgstr "Adresse IP invalide"
+
+msgid "Invalid profile format"
+msgstr "Format de profil invalid"
```

Comment:
typo in French translation: "invalid" -> "invalide"

Reply:
fixed